clients/fortnite_tracker.py

```diff
@@ -61,14 +62,17 @@ async def _search_username(player_name):
     url = ACCOUNT_SEARCH_URL.format(username=player_name)
 
     async with aiohttp.ClientSession() as client:
-        async with client.get(url, headers=HEADERS) as r:
-            assert r.status == 200
-            r = await r.json()
+        async with client.get(url, headers=HEADERS, allow_redirects=False) as r:
+            assert r.status == 302
+            r = r.headers
 
     if not r:
         raise UserDoesNotExist("Username not found in FN Tracker")
 
-    return r[0]["name"]
+    name = r['location'].split("/")
```

Comment:
Does this work?
```
return unquote(r["Location"][-1])
```

Is it `location` or `Location`?

<img width="772" alt="image" src="https://user-images.githubusercontent.com/7246828/112703930-f2b14400-8e55-11eb-858d-0815134797dd.png">

Reply:
As in `len(name) - 1` vs `-1` (just take from the back without calculations)

Reply:
I do not know python lool
yes that seems like it worked , but not with splice - I still needed to use split

Reply:
Whoops, I meant to say `split` (what you had) with the only difference being `len(name) - 1` vs `-1` haha